commands/migratelxc.go

```diff
@@ -346,3 +370,211 @@ func renameLXDContainers(
 	}
 	return group.Wait()
 }
+
+// startLXDContainers starts the LXD containers that have
+// just been migrated from LXC, or were already migrated
+// but not yet started.
+func startLXDContainers(
+	lxcByHost map[*state.Machine][]*state.Machine,
+	lxdByHost map[*state.Machine]map[string]*lxdContainer,
+	containerNames map[*state.Machine]containerNames,
+) error {
+	running := make(map[string]bool) // keyed by LXD container name
```

Comment:
`set.Strings` here? Not much difference since you're only doing doing contains checks below.